File: 0001-two-sum/0001-two-sum.py
# 1. 배열 index 번호 0부터 마지막까지 순회해서 하나씩 뽑기
# 2. 뽑은 index 번호를 +1씩 하여 taget이 될 수 있는지 비교
# 3. index 번호가 마지막일 경우 종료
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        answer = []
        answer = []
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                if nums[i] + nums[j] == target:
                    return [i, j]
             

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] ans = new int[2];
        // nums for문으로 돌면서 더했을 때 target되면 인덱스 반환
        for (int i = 0; i < nums.length;  i++) {
            for (int j = i + 1; j < nums.length; j++) {
                if (nums[i] + nums[j] == target) {
                    ans[0] = i;
                    ans[1] = j;
                }
                
            }
        }
        return ans;
    }
}